<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandChangeScreenTone</class>
 <widget class="QDialog" name="DialogCommandChangeScreenTone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change screen tone...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
         <item row="0" column="0">
          <widget class="QLabel" name="labelRed">
           <property name="text">
            <string>Red:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="horizontalSliderRed">
           <property name="minimum">
            <number>-255</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="PanelPrimitiveValue" name="panelPrimitiveRed" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelGreen">
           <property name="text">
            <string>Green:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="horizontalSliderGreen">
           <property name="minimum">
            <number>-255</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="PanelPrimitiveValue" name="panelPrimitiveGreen" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelBlue">
           <property name="text">
            <string>Blue:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="horizontalSliderBlue">
           <property name="minimum">
            <number>-255</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="PanelPrimitiveValue" name="panelPrimitiveBlue" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelGrey">
           <property name="text">
            <string>Grey:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="horizontalSliderGrey">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="PanelPrimitiveValue" name="panelPrimitiveGrey" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1">
         <item>
          <widget class="QCheckBox" name="checkBoxColor">
           <property name="text">
            <string>Adding color ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxColor">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>+</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>-</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="PanelPrimitiveValue" name="panelPrimitiveColor" native="true">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>68</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxWaitEnd">
         <property name="text">
          <string>Wait the end of the change before the next command</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0">
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>58</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelTime">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PanelPrimitiveValue" name="panelPrimitiveTime" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelSeconds">
           <property name="text">
            <string>second(s)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandChangeScreenTone</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandChangeScreenTone</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
